stream_buffer.c

```diff
@@ -1000,10 +1006,16 @@ static size_t prvReadMessageFromBuffer( StreamBuffer_t * pxStreamBuffer,
         xNextMessageLength = xBufferLengthBytes;
     }
 
+    /* Use the minimum of the wanted bytes and the available bytes. */
+    xCount = configMIN( xNextMessageLength, xBytesAvailable );

```

Comment:
What is the purpose of adding this here?

Reply:
That min calculation is lifted outside of its original location in `prvReadBytesFromBuffer()`.
https://github.com/FreeRTOS/FreeRTOS-Kernel/blob/81f5892105c4d1c93d2941d6ee6c157cfe70bc5c/stream_buffer.c#L1187-L1188
`prvReadBytesFromBuffer()` was changed to just take a single `xCount` argument, rather than both `xNextMessageLength`, `xBytesAvailable`. This simplifies the other two calls to `prvReadBytesFromBuffer()`.